common/browser.go

```diff
@@ -225,10 +224,12 @@ func (b *Browser) onAttachedToTarget(ev *target.EventAttachedToTarget) {
 	b.logger.Debugf("Browser:onAttachedToTarget", "sid:%v tid:%v bctxid:%v",
 		ev.SessionID, ev.TargetInfo.TargetID, ev.TargetInfo.BrowserContextID)
 
-	var (
-		targetPage = ev.TargetInfo
-		browserCtx = b.getDefaultBrowserContextOrByID(targetPage.BrowserContextID)
-	)
+	targetPage := ev.TargetInfo
+
+	browserCtx, err := b.getDefaultBrowserContextOrMatchedID(targetPage.BrowserContextID)
+	if err != nil {
+		k6ext.Panic(b.ctx, "attaching target to browserContext: %v", err)
```

Comment:
WDYT? To be in line with other error messages.

```suggestion
		k6ext.Panic(b.ctx, "attaching target to browser context: %v", err)
```

Reply:
I've reverted the code so we don't panic now, thanks for the suggestion though 👍

Reply:
:) Sure thing. Glad we got panic earlier. 🙇 to you both!